pandas/core/indexes/datetimes.py

```diff
@@ -1922,6 +1925,48 @@ def tz_convert(self, tz):
         ------
         TypeError
             If DatetimeIndex is tz-naive.
+
+        See Also
+        --------
+        tz_localize : Localize tz-naive DatetimeIndex to given time zone,
+                      or remove timezone from tz-aware DatetimeIndex.
+
+        Examples
+        --------
+        With the `tz` parameter, we can change the DatetimeIndex
+        to other time zones:
+
+        >>> dti = pd.DatetimeIndex(start='2014-08-01 09:00',
+        ...                       freq='H', periods=3)
+
+        >>> dti
+        DatetimeIndex(['2014-08-01 09:00:00',
+                       '2014-08-01 10:00:00',
+                       '2014-08-01 11:00:00'],
+                        dtype='datetime64[ns]', freq='H')
+
+        >>> dti.tz_localize('Europe/Berlin').tz_convert('US/Eastern')
```

Comment:
I wouldn't use tz_convert here, this is about the tz_localize doc-string

Reply:
Updated